src/Symfony/Bridge/Doctrine/Types/UlidType.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Types;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\ConversionException;
+use Doctrine\DBAL\Types\GuidType;
+use InvalidArgumentException;
+use Symfony\Component\Uid\AbstractUid;
+use Symfony\Component\Uid\Ulid;
+
+final class UlidType extends GuidType
+{
+    private const NAME = 'ulid';
+
+    /**
+     * @throws ConversionException
+     */
+    public function convertToPHPValue($value, AbstractPlatform $platform)
+    {
+        if (null === $value || '' === $value) {
+            return null;
+        }
+
+        if ($value instanceof AbstractUid) {
+            return $value;
+        }
+
+        try {
+            return Ulid::fromString($value);
+        } catch (InvalidArgumentException $e) {
```

Comment:
should be `\InvalidArgumentException` and the `use` above should be removed (same in the other file)